src/rabbitmq.js

```diff
@@ -132,7 +139,8 @@ class RabbitMQ {
       }
       const stringContent = JSON.stringify(content)
       const bufferContent = new Buffer(stringContent)
-      return Promise.resolve(this.channel.sendToQueue(queue, bufferContent))
+      return Promise
+        .resolve(this.publishChannel.sendToQueue(queue, bufferContent))
```

Comment:
quick question. I was looking through the docs and from the example they gave it looks like it waits for the message to be ack'd before it completes?
```
require('amqplib/callback_api').connect(function(err, c) {
  c.createConfirmChannel(function(err, ch) {
    ch.sendToQueue('foo', new Buffer('foobar'), {}, function(err, ok) {
      if (err !== null) console.warn('Message nacked!');
      else console.log('Message acked');
    });
  });
});
```

do you know if that means it wait for the jobs `ack` or is that some other kind of `ack` it waits for?

http://www.squaremobius.net/amqp.node/channel_api.html#confirmchannel